rich-default/environment.yml

```diff
@@ -0,0 +1,522 @@
+name: base
+channels:
+  - samoturk
+  - mordred-descriptor
+  - rdkit
+  - conda-forge
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - _r-mutex=1.0.1=anacondar_1
+  - aiohttp=3.6.2=py36h516909a_0
```

Comment:
I think it is sufficient for our case to remove the =py36...

Maybe try building with only pinning version numbers of packages. The stuff after the second equals are build identifers and it may be ok for us to ignore them for our purposes.

Reply:
I retrieved `environment.yml` using this command, which removes build identifiers.
`conda env export -n base > environment.yml --no-builds`